posts/2022-01-31-configuring-ocp-for-user-namespace.md

```diff
@@ -0,0 +1,198 @@
+  ```
+
+  ```raw
+  runc version 1.0.3
+  spec: 1.0.2-dev
+  go: go1.17.2
+  libseccomp: 2.5.1
+  ```
+
+  ```sh
+  ssh -i ~/.crc/machines/crc/id_ecdsa core@192.168.130.11 journalctl -u install-runc.service
+  ```
+
+  ```raw
+  -- Logs begin at Sat 2021-12-11 13:38:56 UTC, end at Wed 2022-01-26 07:12:23 UTC. --
+  Jan 26 06:50:13 crc-hsl9k-master-0 bash[1658]: package runc-1.0.3-992.rhaos4.10.el8.x86_64 is not installed
+  Jan 26 06:50:13 crc-hsl9k-master-0 systemd[1]: Started Install custom runc.
+  Jan 26 06:50:14 crc-hsl9k-master-0 bash[1658]: Downloading 'https://ftweedal.fedorapeople.org/runc-1.0.3-992.rhaos4.10.el8.x86_64.rpm'... done!
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Checking out tree 26d80bc...done
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: No enabled rpm-md repositories.
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Importing rpm-md...done
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Resolving dependencies...done
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Applying 1 override and 5 overlays
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Processing packages...done
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Running pre scripts...done
+  Jan 26 06:50:16 crc-hsl9k-master-0 bash[1658]: Running post scripts...done
+  Jan 26 06:50:17 crc-hsl9k-master-0 bash[1658]: Running posttrans scripts...done
+  Jan 26 06:50:17 crc-hsl9k-master-0 bash[1658]: Writing rpmdb...done
+  Jan 26 06:50:18 crc-hsl9k-master-0 bash[1658]: Writing OSTree commit...done
+  Jan 26 06:50:19 crc-hsl9k-master-0 bash[1658]: Staging deployment...done
+  Jan 26 06:50:20 crc-hsl9k-master-0 systemd[1]: Stopping Install custom runc...
+  Jan 26 06:50:20 crc-hsl9k-master-0 systemd[1]: install-runc.service: Succeeded.
+  Jan 26 06:50:20 crc-hsl9k-master-0 systemd[1]: Stopped Install custom runc.
+  Jan 26 06:50:20 crc-hsl9k-master-0 systemd[1]: install-runc.service: Consumed 94ms CPU time
+  -- Reboot --
+  Jan 26 06:51:10 crc-hsl9k-master-0 bash[1656]: runc-1.0.3-992.rhaos4.10.el8.x86_64
+  Jan 26 06:51:09 crc-hsl9k-master-0 systemd[1]: Started Install custom runc.
+  Jan 26 06:51:09 crc-hsl9k-master-0 systemd[1]: install-runc.service: Succeeded.
+  Jan 26 06:51:09 crc-hsl9k-master-0 systemd[1]: install-runc.service: Consumed 11ms CPU time
+  ```
+
+- For the cgroup2:
+
+  ```sh
+  ssh -i ~/.crc/machines/crc/id_ecdsa core@192.168.130.11 mount | grep cgroup2
+  ```
+
+- For the kernelarguments:
+
+  ```sh
+  ssh -i ~/.crc/machines/crc/id_ecdsa core@192.168.130.11 cat /proc/cmdline
+  BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-36fd944867b0e491991a65f6f3b7209c937fe3bd7cdbd855c7c5d5a7070ce570/vmlinuz-4.18.0-305.28.1.el8_4.x86_64 random.trust_cpu=on console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu ostree=/ostree/boot.1/rhcos/36fd944867b0e491991a65f6f3b7209c937fe3bd7cdbd855c7c5d5a7070ce570/0 root=UUID=91ba4914-fd2b-4a7c-b498-28585a80a40e rw rootflags=prjquota systemd.unified_cgroup_hierarchy=1 cgroup_no_v1=all psi=1
+  ```
```

Comment:
Alternatively, you could observe the cgroupv2 unified heirarchy under `/sys/fs/cgroup`.

Reply:
Thanks! I have added into the previous bullet list item.